org.fritzing.Fritzing.json

```diff
@@ -0,0 +1,100 @@
+{
+    "id": "org.fritzing.Fritzing",
+    "runtime": "org.kde.Platform",
+    "sdk": "org.kde.Sdk",
+    "runtime-version": "5.9",
+    "branch": "stable",
+    "command": "Fritzing",
+    "rename-desktop-file": "fritzing.desktop",
+    "rename-appdata-file": "fritzing.appdata.xml",
+    "rename-icon": "fritzing",
+    "finish-args": [
+        "--socket=x11", "--share=ipc",
+        "--socket=wayland",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-documents",
+        "--share=network"
+    ],
+    "cleanup": [
+        "*.a",
+        "*.la",
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/cmake",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh",
+                "./b2 install variant=release link=shared runtime-link=shared --prefix=/app -j $FLATPAK_BUILDER_N_JOBS"
```

Comment:
You can limit which boost libraries are being built to shorten the build time (and make the app smaller) with `--with-libraries=`.